idd/Energy+.idd.in

```diff
@@ -79692,8 +79763,42 @@ FaultModel:Fouling:AirFilter,
        \object-list CubicCurves
        \object-list UniVariateTables
 
+FaultModel:Fouling:CoolingTower,
+   \memo This object describes the fault of fouling cooling towers
+   \min-fields 6
+   A1, \field Name
+       \note Enter the name of the fault
+       \required-field
+       \type alpha
+   A2, \field Availability Schedule Name
+       \type object-list
+       \object-list ScheduleNames
+   A3, \field Severity Schedule Name
+       \type object-list
+       \object-list ScheduleNames
+   A4, \field Cooling Tower Object Type
+       \note Enter the type of the cooling tower affected
+       \required-field
+       \type choice
+       \key CoolingTower:SingleSpeed
+       \key CoolingTower:TwoSpeed
+       \key CoolingTower:VariableSpeed:MERKEL
+   A5, \field Cooling Tower Object Name
```

Comment:
Mike, CoolingTower:VariableSpeed is not listed here on purpose. Different from the other three tower models, this tower model (CondenserLoopTowers::CalcVariableSpeedTower) use empirical curves instead of tower U-Factor for the tower calculations. Therefore, the designed fouling fault model cannot be applied to it.